<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistryGroupWidget</class>
 <widget class="QWidget" name="RegistryGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #121212;
}

QGroupBox {
	font: 75 16pt &quot;Century Gothic&quot;;
	color: #d9d9d9;
}

QLabel {
	font: 75 24pt &quot;Century Gothic&quot;;
	color: #d9d9d9;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Registers</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>292</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,1,1,1,1,1">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="xRegistryLabel">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="x_hex_registry">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="pc_hex_registry">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pcRegistryLabel">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>PC</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="cc_hex_registry">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ccRegistryLabel">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>CC</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="x_bin_registry">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="a_hex_registry">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="a_bin_registry">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="spRegistryLabel">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>SP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="bin_label">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 14pt &quot;Century Gothic&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>BINARY</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="yRegistryLabel">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="hex_label">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 14pt &quot;Century Gothic&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>HEX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="sp_hex_registry">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="sp_bin_registry">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="y_hex_registry">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="pc_bin_registry">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="y_bin_registry">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="aRegistryLabel">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="cc_bin_registry">
       <property name="font">
        <font>
         <family>Digital-7 Mono</family>
         <pointsize>28</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 75 28pt &quot;Digital-7 Mono&quot;;
	color: #d9d9d9;
}</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
